requirements/edx/base.in

```diff
@@ -93,7 +93,7 @@ futures ; python_version == "2.7"   # via django-pipeline, python-swift-client,
 glob2==0.3                          # Enhanced glob module, used in openedx.core.lib.rooted_paths
 gunicorn==17.5
 help-tokens
-html5lib==0.999                     # HTML parser, used for capa problems
+html5lib==1.0.1                     # HTML parser, used for capa problems
```

Comment:
Since this is the current release, could you just remove the version constraint?  That would let us automatically notice and test new versions via `make upgrade`, while still keeping the version pinned in `base.txt`, etc.